webhooks-extension/src/components/WebhookCreate/WebhookCreate.js

```diff
@@ -86,10 +85,8 @@ class WebhookCreatePage extends Component {
   }
 
   async fetchPipelines(namespace) {
-    let pl;
     try {
-      pl = await getPipelines(namespace);
-      this.setState({apiPipelines: pl})
+      this.props.fetchPipelines(namespace);
     } catch (error) {
```

Comment:
the error should already be caught by the action and is exposed via the `getPipelinesErrorMessage` selector, no need for the try-catch here

Reply:
done. Added logic to output notification if error happens and a test case.